demo/components/presentational/Land.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ListView, Image, StyleSheet, View, Text } from "react-native";
+
+export default class ListLands extends React.Component {
+  renderRow = rowData => {};
+
+  render() {
+    const { id, name, img, priceMana, priceUsd } = this.props.land;
+    return (
+      <View style={styles.landContainer}>
+        <Image source={img} />
+        <Text>{name}</Text>
+        <Text>
+          ⏣ {priceMana} (~ ${priceUsd})
```

Comment:
We probably want the word MANA and a $ sign here

Reply:
MANA 123 (~ $10)
or
MANA $123 (~ $10)

Reply:
I think I would go with `123 MANA (~$10)`. This reminds me that we'll want to build things in a way that sets us up well for internationalization.